src/sinks/azure_blob/integration_tests.rs

```diff
@@ -170,9 +181,13 @@ async fn azure_blob_insert_json_into_blob_gzip() {
     assert_eq!(blobs.len(), 1);
     assert!(blobs[0].clone().ends_with(".log.gz"));
     let (blob, blob_lines) = config.get_blob(blobs[0].clone()).await;
+    assert_eq!(
+        blob.properties.content_encoding,
+        Some(String::from("gzip"))
+    );
     assert_eq!(
         blob.properties.content_type,
-        String::from("application/gzip")
+        String::from("application/json")
```

Comment:
I *think* the content-type here will still be `application/x-ndjson` due to the newline delimited framer in the encoder.
```suggestion
        String::from("application/x-ndjson")
```

Reply:
Good callout, overlooked this and the related test was ignored.